.vscode/settings.json

```diff
@@ -1,6 +1,7 @@
 {
     "cSpell.words": [
         "apisports",
+        "Hola",
```

Comment:
Is there a particular reason behind adding these words here? Maybe if linter annoys you I guess. 
By the way, `.vscode` folder should be included in `.gitignore`, there are your local project settings

Reply:
Yes, I'm used to adding these so just for the sake of not having all these underlined words, just added it to .gitignore